LINUX_keep_current.md

```diff
@@ -0,0 +1,285 @@
+
+# Keeping your setup current
+
+This section contains the steps that you need to follow in order to make sure that your setup is up to date
+
+First and foremost, in order to work in good conditions, make sure that :
+- you are working with a high speed internet connection
+- your computer has enough memory (8GB), so that you can run your code efficiently
+- your computer has enough disk space (30GB), so that you can work on big datasets
+
+## git
+
+Verify that git works
+
+``` bash
+git --version
+```
+
+👉 You should expect an output similar to this one yielding the git version :
+
+``` bash
+git version 2.33.0
```

Comment:
do you really want to update this line every time git change version?

Reply:
cc @gmanchon

Reply:
the idea behind the wording `similar to this one` was to avoid following the version changes, while giving the student an idea of what is expected...

Reply:
maybe we could specify something like git > 2.xxx wdyt ?